ckan/templates-midnight-blue/home/snippets/featured_group.html

```diff
@@ -1,7 +1,13 @@
-{% set groups = h.get_featured_groups() %}
-
-{% for group in groups %}
-  <div class="card box">
-    {% snippet 'snippets/group_item.html', group=group %}
-  </div>
-{% endfor %}
+{% set groups = h.get_featured_groups(count=6) %}
+<h2>{{ _("Groups") }}</h2>
+<p>{{ _("Browse groups") }}</p>
```

Comment:
I'm not sure these sub-headers add much in their current form (Groups / Browse groups, Organizations /Browse organizations...). Either we replace them with something generic like "Browse dataset by categories" / "Browse data publishers" (which might still be not relevant in certain instances) or we dropped them.

@ckan/core what do you think?

![Screenshot 2025-01-14 at 12-12-01 Welcome - CKAN](https://github.com/user-attachments/assets/7c79a6c1-a872-4772-9682-3d4237e89fa1)

Reply:
I've removed them for now, if there is a desire to add them back in I will!